packages/cli/generators/relation/templates/controller-relation-template-has-one.ts.ejs

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
The template should be similar to `controller-relation-template-has-many.ts.ejs`. I think it's fair to has methods such as `create` and `find` in the relation controller. And to add these endpoints to the template, fields `targetModelPath` and `targetModelRequestBody` need to be defined in `has-one-relation-generator.ts`([Ref](https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/relation/has-many-relation.generator.js#L41)).